### PR TITLE
SIM-1433 ADD release 3.2.0 anouncement

### PR DESCRIPTION
**Pull request by mgh87**

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,6 +6,15 @@ title: jSparrow
 
 ![jSparrow Linebreak Very-Top](/dashboard/img/git-linebreak-very-top.png)
 
+## jSparrow 3.2.0 released!!!
+
+On Thursday, 21 February 2019, jSparrow 3.2.0 release arrived.
```
Comment (luigiwerzowa) [inline docs/README.md]: Thursday, February 21, 2019,

Comment (mgh87) [reply docs/README.md]: Ardit suggested this style because it was mentioned somewhere :dancer:

Comment (mgh87) [reply docs/README.md]:
```suggestion
On Thursday, 21 February, 2019, jSparrow 3.2.0 release arrived. 
```

Comment (mgh87) [reply docs/README.md]: When we add suggestions, we can edit them within the github.com UI and resolve them.

Comment (mgh87) [reply docs/README.md]:
```suggestion
On Thursday, February 21 2019, jSparrow 3.2.0 release arrived. 
```

Ludwig mentioned that I overlooked the reordering *wink*

Comment (luigiwerzowa) [reply docs/README.md]: The order is important.

File: docs/README.md [hunk 2]
```diff
@@ -6,6 +6,15 @@ title: jSparrow
 
 ![jSparrow Linebreak Very-Top](/dashboard/img/git-linebreak-very-top.png)
 
+## jSparrow 3.2.0 released!!!
+
+On Thursday, 21 February, 2019, jSparrow 3.2.0 release arrived.
```
Comment (mgh87) [inline docs/README.md]:
```suggestion
On Thursday, February 21, 2019, jSparrow 3.2.0 release arrived. 
```

Ludwig mentioned that I overlooked the reordering *wink*

**Pull request merged**